src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/SignInServiceTests.cs

```diff
@@ -26,685 +26,387 @@ public class SignInServiceTests
-                .ReturnsAsync(userPublicationInvites);
-            userPublicationInviteRepository
-                .Setup(mock => mock.RemoveByUserEmail(email, It.IsAny<CancellationToken>()))
-                .Returns(Task.CompletedTask);
-
-            var userReleaseRoleRepository = new Mock<IUserReleaseRoleRepository>(Strict);
-            foreach (var userReleaseInvite in userReleaseInvites)
-            {
-                userReleaseRoleRepository
-                    .Setup(mock =>
-                        mock.Create(
-                            It.IsAny<Guid>(),
-                            userReleaseInvite.ReleaseVersionId,
-                            userReleaseInvite.Role,
-                            CreatedById
-                        )
-                    )
-                    .ReturnsAsync(new UserReleaseRole());
-            }
-
-            var userPublicationRoleRepository = new Mock<IUserPublicationRoleRepository>(Strict);
-            foreach (var userPublicationInvite in userPublicationInvites)
-            {
-                userPublicationRoleRepository
-                    .Setup(mock =>
-                        mock.Create(
-                            It.IsAny<Guid>(),
-                            userPublicationInvite.PublicationId,
-                            userPublicationInvite.Role,
-                            CreatedById
-                        )
-                    )
-                    .ReturnsAsync(new UserPublicationRole());
-            }
-
-            await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
-            await using (var usersAndRolesDbContext = InMemoryUserAndRolesDbContext(usersAndRolesDbContextId))
-            {
-                var service = SetupService(
-                    contentDbContext: contentDbContext,
-                    usersAndRolesDbContext: usersAndRolesDbContext,
-                    userService: userService.Object,
-                    userManager: userManager.Object,
-                    userReleaseInviteRepository: userReleaseInviteRepository.Object,
-                    userPublicationInviteRepository: userPublicationInviteRepository.Object,
-                    userReleaseRoleRepository: userReleaseRoleRepository.Object,
-                    userPublicationRoleRepository: userPublicationRoleRepository.Object
-                );
-
-                var result = await service.RegisterOrSignIn();
-                result.AssertRight();
-            }
-
-            VerifyAllMocks(
-                userService,
-                userManager,
-                userReleaseInviteRepository,
-                userPublicationInviteRepository,
-                userReleaseRoleRepository,
-                userPublicationRoleRepository
+                }
+            )
+            .Generate();
+
+        var userReleaseInvites = _dataFixture
+            .DefaultUserReleaseInvite()
+            .WithEmail(userWithExpiredInvite.Email)
```

Comment:
Looks like `userReleaseInvites` is redundant now, but should it be?

I think we should be saving these, otherwise we can't test they've been removed effectively?